src/Sulu/Bundle/AdminBundle/Resources/js/containers/List/adapters/TreeTableAdapter.js

```diff
@@ -6,16 +6,23 @@ import Table from '../../../components/Table';
 import Loader from '../../../components/Loader';
 import TreeStructureStrategy from '../structureStrategies/TreeStructureStrategy';
 import FullLoadingStrategy from '../loadingStrategies/FullLoadingStrategy';
+import Pagination from '../../../components/Pagination';
+import PaginatedLoadingStrategy from '../loadingStrategies/PaginatedLoadingStrategy';
+import type {LoadingStrategyInterface} from '../types';
 import AbstractTableAdapter from './AbstractTableAdapter';
 
 @observer
 class TreeTableAdapter extends AbstractTableAdapter {
-    static LoadingStrategy = FullLoadingStrategy;
-
     static StructureStrategy = TreeStructureStrategy;
 
+    static paginatable = true;
```

Comment:
not needed anymore when passinge `paginated` to the constructor of the `LoadingStrategy`